flask_swagger_generator/utils/url_variable_type.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum
+from flask_swagger_generator.exceptions import SwaggerGeneratorException
+
+
+class UrlVariableType(Enum):
+    INTEGER = 'integer'
+    STRING = 'string'
+    FLOAT = 'float'
+    PATH = 'path'
+    UUID = 'uuid'
+
+    @staticmethod
+    def from_string(value: str):
+
+        if isinstance(value, str):
+
+            if value.lower() == 'string':
+                return UrlVariableType.STRING
+            elif value.lower() == 'int':
+                return UrlVariableType.INTEGER
+            elif value.lower() == 'float':
```

Comment:
Float, path and uuid types are not recognized by swagger although these can be added in the url. So these should not be used to resolve input type in SwaggerPathParameter class. We can add some mapping for url variable type to swagger path types. 
The only allowed values by swagger are: **array, boolean, integer, number, object, string**

We can use these mappings:
```
float : number
path : string
uuid : string (not sure about this one, I will need to look into this)
```

Reply:
I removed the references to uuid, path and float

```python
def get_flask_url_variable_type_value(self):

    if self == UrlVariableType.STRING:
        return 'string'
    elif self == UrlVariableType.INTEGER:
        return 'integer'
    elif self == UrlVariableType.FLOAT:
        return 'number'
    elif self == UrlVariableType.PATH:
        return 'string'
    elif self == UrlVariableType.UUID:
        return 'string'
```

Reply:
Could you review the changes again